e2e/specs/problem-list.e2e-spec.ts

```diff
@@ -54,11 +54,12 @@ describe('Problem list tests', () => {
             await navigationFlow.goToProblemsPage();
         });
 
-        afterEach(async () => {
```

Comment:
is this needed? I think that the new reported includes error msgs.

Reply:
I honestly don't know :) Didn't want to just throw it away though